include/platform/mir/graphics/platform.h

```diff
@@ -131,9 +159,10 @@ typedef mir::UniqueModulePtr<mir::graphics::RenderingPlatform>(*CreateRenderPlat
 typedef void(*AddPlatformOptions)(
     boost::program_options::options_description& config);
 
-typedef mir::graphics::PlatformPriority(*PlatformProbe)(
+typedef std::vector<mir::graphics::SupportedDevice>(*PlatformProbe)(
     std::shared_ptr<mir::ConsoleServices> const&,
-        mir::options::ProgramOption const& options);
+    std::shared_ptr<mir::udev::Context> const&,
+    mir::options::ProgramOption const& options);
```

Comment:
I think, inter alia, this is a platform ABI break: `MIR_SERVER_GRAPHICS_PLATFORM_ABI` needs bumping

Reply:
You're right; this *does* break platform ABI (and `MIR_SERVER_GRAPHICS_PLATFORM_ABI` needs bumping), but I don't think this *does* change `libmirplatform` ABI (and hence we don't need to change SONAME).

Reply:
In further reflection you're right. We don't actually use the changes in that ABI.